data/xfstests/wrapper.sh

```diff
@@ -2,14 +2,18 @@
 XFSTESTS_DIR='/opt/xfstests'
 PROG="$XFSTESTS_DIR/check"
 SCRIPT_DIR=$(realpath $(dirname "$0"))
-if [ "$#" -gt 2 ]; then
-    INJECT_LINE=$2
-    INJECT_CODE=$3
+OPTIONS=""
+if [ "$#" -gt 1 ]; then
+    OPTIONS=$2
+fi
+if [ "$#" -gt 3 ]; then
```

Comment:
This line should sync with line 87 of original file. But IMO use one sentence is more elegant.

Reply:
Added more verify scenarios. In the 87th line, it used $1 not affected by this options(which is $2), and the following parameters are all with para_names, so not affected too. But I do missing the parameter for inject_info, I add it in a new commit.